src/components/pill/container.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components"
+import Flex from "src/components/templates/flex"
+import { getColor } from "src/theme"
+
+const getBackground = ({ theme, background, flavour = "neutral", hollow }) => {
+  if (background) return
+  const type = hollow ? "hollow" : "background"
+  const value = getColor(["pill", type, flavour])({ theme })
+  return `background-color: ${value};`
+}
+
+const Container = styled(Flex).attrs(({ round = 999, hollow, flavour, borderColor }) => ({
+  padding: [1, 2],
+  round,
+  border: {
+    side: "all",
+    color: borderColor ? borderColor : ["pill", hollow ? "border" : "background", flavour],
+    size: "1px",
+  },
```

Comment:
Will we need also borderless?
![image](https://user-images.githubusercontent.com/5786722/117157034-86175680-adbe-11eb-8676-0ca801c7cb8b.png)

Reply:
I would keep the border, to keep all the pills consistent, but just make it to the `hollow` color when `borderless` prop is present. wdyt?